.devcontainer/devcontainer.json

```diff
@@ -15,7 +15,7 @@
 	// "features": {},
 
 	// For use with PHP or Apache (e.g.php -S localhost:8080 or apache2ctl start)
-	"forwardPorts": [3306, 8080],
+	//"forwardPorts": [3306, 8080],
```

Comment:
May this commented out line need to be uncommented at some point? Otherwise, please remove it.

Reply:
Done